src/server.c

```diff
@@ -7155,7 +7157,7 @@ int main(int argc, char **argv) {
     serverLog(LL_NOTICE, "oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo");
     serverLog(LL_NOTICE,
         "Redis version=%s, bits=%d, commit=%s, modified=%d, pid=%d, just started",
-            REDIS_VERSION,
+            SERVER_VERSION,
```

Comment:
Shouldn't we modify the log line to indicate it's `Valkey version` ?

Reply:
I was trying to avoid touching too much stuff, since I know there are other PRs floating around.